app/src/main/java/com/iic/mokojin/PlayerQueueFragment.java

```diff
@@ -49,6 +53,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             public EnhancedListView.Undoable onDismiss(final EnhancedListView enhancedListView, int i) {
                 try {
                     new LeaveQueueOperation().run((QueueItem) mQueueListView.getItemAtPosition(i));
+                    mQueueAdapter.dismissItem(i);
+                    mQueueAdapter.notifyDataSetChanged();
```

Comment:
nope. `LeaveQueueOperation` does.

see below for `dismissItem`. It does a sort of "local dismiss" to make the adapter act as if the dismissed item no longer exists (even though it does, until the adapter gets the changes from the server)